lms/static/styles/components/_FilePickerApp.scss

```diff
@@ -31,9 +27,26 @@
   top: calc(50% - #{var.$spinner-size / 2});
 }
 
-.FilePickerApp__document-source-buttons {
-  align-items: stretch;
-  display: inline-flex;
-  flex-direction: column;
-  margin-top: 30px;
+.FilePickerApp__actions {
+  // A flex container (row) with two children: a column of buttons and a
+  // "spacer" to take up the rest of the row space not needed by the buttons.
+  // The spacer prevents the button container from stretching full width.
+  display: flex;
+  // Add some white space before we start rendering the buttons
+  margin-top: 3em;
+
+  &-buttons {
+    // Display buttons in a column with a vertical margin between each button
+    display: flex;
+    flex-direction: column;
+    & > * + * {
```

Comment:
A possible way to achieve the same result without `!important`:

```scss
& > :not(:first-child) {
  margin-top: 0.5em;
}
```

Reply:
This is a very interesting comment, thank you. A simple way to bump up specificity. Might I suggest:

* We leave this as-is for the moment, as the `!important` is caged in pretty well and is probably _safe_, and
* I have an issue where I'm tracking some work related to the existing layout utility classes and mixins: https://github.com/hypothesis/frontend-shared/issues/74 . This selector/rule is based on the `vertical-rhythm` mixin/pattern. I could evaluate at that time using this higher-specificity strategy for those mixins and utility classes.

In summary, I expect these new rules (here, in this LMS component's SASS) to have a short lifespan, to be (aspirationally) replaced by shared patterns soon.

Reply:
Because this is going to be replaced shortly, I'm in favor of just leaving this. We may want to include Rob's suggestion in the util class in the shared  lib.